kolibri/plugins/learn/assets/src/views/LibraryPage/AccessibleFolderCard.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+
+  <div>
+    <ul>
+      <KCard
+        class="card"
+        :to="to"
+        :headingLevel="2"
+        layout="horizontal"
+        thumbnailDisplay="large"
+        :title="contentNode.title"
+      >
+        <template #thumbnailPlaceholder>
+          <div>
+            <CardThumbnail
```

Comment:
**[required]** Regarding the usage of `<CardThumbnail>` here, `KCard` doesn't need to use another component to re-implement the whole thumbnail area because it already provides it. If we used `<CardThumbnail>` here, that would override everything in unintended ways, and it's also unnecessary extra work. Let's leave `KCard` to do everything it can and only put folder icon into the `#thumbnailPlaceholder` slot. I realize that the slot's name is not intuitive in this regard - can you think of a better name?

(side note for context, ultimately, a big future goal is to get rid of all current card components in Kolibri, `<CardThumbnail>` included, in favor of `KCard`)